vico/views/src/main/java/com/patrykandpatrick/vico/views/cartesian/CartesianChartView.kt

```diff
@@ -73,7 +74,7 @@ constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
       ranges = CartesianChartRanges.Empty,
       scrollEnabled = false,
       zoomEnabled = false,
-      layerPadding = themeHandler.chart?.layerPadding ?: CartesianLayerPadding(),
+      layerPadding = themeHandler.chart?.layerPadding?.invoke(extraStore) ?: CartesianLayerPadding(),
```

Comment:
`CartesianChartModel.extraStore` isn’t available at this stage, but an initial value is required. `CartesianLayerPadding()` works; it’ll be replaced before drawing. (Down the road, it may be beneficial to introduce a `CartesianLayerPadding.Zero` singleton, but this needn’t be handled in this PR.)

```suggestion
      layerPadding = CartesianLayerPadding(),
```